src/main/java/org/sonatype/repository/helm/internal/hosted/HostedHandlers.java

```diff
@@ -42,7 +42,7 @@
 
   @Inject
   public HostedHandlers(final HelmPathUtils helmPathUtils) {
-    this.helmPathUtils = checkNotNull(helmPathUtils);
+    this.helmPathUtils = Objects.requireNonNull(helmPathUtils);
```

Comment:
I would switch to using `import static com.google.common.base.Preconditions.checkNotNull;` as @bhamail pointed out, that's consistent with Nexus Repo